src/containers/Charts/Chart.js

```diff
@@ -139,9 +137,9 @@ const Chart = ({ solarData, history }) => {
               required
               onChange={e => getSpecificChart(e)}
             >
-              <option id='Solar' data-y-axis='(kWh / m^2) / day' value="solrad_monthly">Solar Radiation</option>
-              <option id='AC' data-y-axis='kWh' value="ac_monthly">AC Monthly</option>
-              {solarData.savings && <option id='Dollars $' data-y-axis='' value="savings">Savings Value</option>}
+              <option id='Solar' data-y-axis='(kWh / m^2) / day' value="solRadMonthly">Solar Radiation</option>
+              <option id='AC' data-y-axis='kWh' value="acMonthly">AC Monthly</option>
+              {solarData.savingsMonthly && <option id='Dollars $' data-y-axis='$' value="savingsMonthly">Savings Value</option>}
```

Comment:
Maybe on the y-axis for this one we could put `Dollars ($)` because it's a little off with just the sideways dollar sign in there

Reply:
Agreed